content/03-reference/01-tools-and-interfaces/07-preview-features/001-sql-server/index.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: 'Microsoft SQL Connector'
+metaTitle: 'Microsoft SQL Connector (Reference)'
+metaDescription: 'A list of Prisma features currently in preview.'
+
+---
+
+<TopBlock>
+
+The Microsoft SQL Connector is available as a preview feature in 2.10.0.
```

Comment:
Suggestion: Link version numbers to GH release notes by default maybe so people can easily get context about that version.